addons/l10n_ca/data/template/account.tax-ca_2023.csv

```diff
@@ -7,9 +7,9 @@
 "","","","","","","","","","","","tax","invoice","+Ca103||+CaQc103","l10n_ca_231000","",""
 "","","","","","","","","","","","base","refund","-Ca101||-CaBcA||-CaQc101","","",""
 "","","","","","","","","","","","tax","refund","-Ca103||-CaQc103","l10n_ca_231000","",""
-"pst_sale_tax_6","6% PST SK","6%","6% PST SK","sale","6.0","percent","20","False","tax_group_pst_6","","base","invoice","","","6% TVP SK","6% TVP SK"
+"pst_sale_tax_6","6% PST SK","6%","6% PST SK","sale","6.0","percent","20","False","tax_group_pst_6","","base","invoice","+CaSk8Base","","6% TVP SK","6% TVP SK"
 "","","","","","","","","","","","tax","invoice","+CaSk8","l10n_ca_232000","",""
-"","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","base","refund","-CaSk8Base","","",""
```

Comment:
```suggestion
"","","","","","","","","","","","base","refund","","","",""
```

Since the worksheet supplement counts credit notes under tax credits, Laura and I think the total sales should not include refunds.

![image](https://github.com/odoo/odoo/assets/74787742/6c0b2d5e-725a-4d28-aabe-b50694bd5971)

Reply:
... She told me the opposite xD
And the worksheet contains what is collected, what is refunded and they actually report the difference on the final report (where the total sales are inputed). So the total sales and the total taxes would not match :thinking: